lib/mrsk/configuration.rb

```diff
@@ -87,6 +84,12 @@ def env_args
     end
   end
 
+  def volumes
+    return unless config.volumes.present?
```

Comment:
Turn into full conditional, like the other usage, so `if config.volumes.present?`.

Reply:
done